osu.Game/Overlays/MusicController.cs

```diff
@@ -243,6 +256,63 @@ private PreviousTrackResult prev()
                 onSuccess?.Invoke();
         });
 
+        /// <summary>
+        /// Attenuates the volume and/or filters the currently playing track.
+        /// </summary>
+        /// <param name="duration">Duration of the ducking transition, in ms.</param>
+        /// <param name="duckVolumeTo">Level to drop volume to (1.0 = 100%).</param>
+        /// <param name="duckCutoffTo">Cutoff frequency to drop `AudioFilter` to. Use `null` to skip filter effect.</param>
+        /// <param name="easing">Easing for the ducking transition.</param>
+        /// <param name="unduckDuration">Duration of the unducking transition, in ms.</param>
+        /// <param name="unduckEasing">Easing for the unducking transition.</param>
+        public IDisposable Duck(int duration = 0, float duckVolumeTo = 0.25f, int? duckCutoffTo = 300, Easing easing = Easing.OutCubic, int unduckDuration = 500, Easing unduckEasing = Easing.InCubic)
```

Comment:
I feel like cubic easing is too abrupt in its current form. Can you give a plain `In`/`Out` a try? It already feels better to me (also a slight reduction to the unduck delay):

```diff
diff --git a/osu.Game/Overlays/MusicController.cs b/osu.Game/Overlays/MusicController.cs
index 8b52c59bae..06845116b1 100644
--- a/osu.Game/Overlays/MusicController.cs
+++ b/osu.Game/Overlays/MusicController.cs
@@ -265,7 +265,7 @@ public void NextTrack(Action onSuccess = null) => Schedule(() =>
         /// <param name="easing">Easing for the ducking transition.</param>
         /// <param name="unduckDuration">Duration of the unducking transition, in ms.</param>
         /// <param name="unduckEasing">Easing for the unducking transition.</param>
-        public IDisposable Duck(int duration = 0, float duckVolumeTo = 0.25f, int? duckCutoffTo = 300, Easing easing = Easing.OutCubic, int unduckDuration = 500, Easing unduckEasing = Easing.InCubic)
+        public IDisposable Duck(int duration = 0, float duckVolumeTo = 0.25f, int? duckCutoffTo = 300, Easing easing = Easing.Out, int unduckDuration = 500, Easing unduckEasing = Easing.In)
         {
             if (audioDuckActive) return null;
 
@@ -292,7 +292,7 @@ public IDisposable Duck(int duration = 0, float duckVolumeTo = 0.25f, int? duckC
         /// <param name="duckCutoffTo">Cutoff frequency to drop `AudioFilter` to. Use `null` to skip filter effect.</param>
         /// <param name="duckDuration">Duration of the ducking transition, in ms.</param>
         /// <param name="duckEasing">Easing for the ducking transition.</param>
-        public void TimedDuck(int delay, int unduckDuration = 500, Easing unduckEasing = Easing.InCubic, float duckVolumeTo = 0.25f, int? duckCutoffTo = 300, int duckDuration = 0, Easing duckEasing = Easing.OutCubic)
+        public void TimedDuck(int delay, int unduckDuration = 500, Easing unduckEasing = Easing.In, float duckVolumeTo = 0.25f, int? duckCutoffTo = 300, int duckDuration = 0, Easing duckEasing = Easing.Out)
         {
             if (audioDuckActive) return;
 
diff --git a/osu.Game/Overlays/Toolbar/ToolbarRulesetSelector.cs b/osu.Game/Overlays/Toolbar/ToolbarRulesetSelector.cs
index 7da6b76aaa..07ce1f69ef 100644
--- a/osu.Game/Overlays/Toolbar/ToolbarRulesetSelector.cs
+++ b/osu.Game/Overlays/Toolbar/ToolbarRulesetSelector.cs
@@ -122,7 +122,7 @@ private void playRulesetSelectionSample(ValueChangedEvent<RulesetInfo> r)
 
             rulesetSelectionChannel[r.NewValue] = channel;
             channel.Play();
-            musicController?.TimedDuck(600);
+            musicController?.TimedDuck(500);
         }
 
         public override bool HandleNonPositionalInput => !Current.Disabled && base.HandleNonPositionalInput;

```

Reply:
Can you elaborate on which usage(s) you find to be too abrupt? If it's just the ruleset selector, probably better to change that specific usage rather than the default?

`CubicIn`/`CubicOut` matches what the previous implementations (`DialogOverlay`, etc) used, so they should sound identical to before.

Reply:
I was testing with the ruleset selector, but I think it might be a better default. I'll check the other usages and loop back.

Reply:
I think non-cubic sounds better for the dialog cases too. There's not much difference since the duration is so short, but it's a little less sharp.